### ISSUE TITLE
SuperTab Chaining Completion Does not Initially Work with vim-latex

### ISSUE THREAD

**Issue by gkapfham** [lervag/vimtex#37]
Thank you for developing vim-latex.  I really like it so far!

When I was previously using Latex-Box, I used the following code in my .vimrc file to enable two types of completion: (1) the standard "next" completion that uses my dictionary and the current buffer to complete words and (2) Latex-Box for items in the bibliography and other labels in the paper.

``` vim
autocmd FileType tex
            \ if &omnifunc != '' |
            \   call SuperTabChain(&omnifunc, "<c-n>") |
            \   call SuperTabSetDefaultCompletionType("<c-x><c-u>") |
            \ endif
```

However, using this "chaining" feature of SuperTab does not seem to work with vim-latex when I type a word llike "exam" and then type "tab".

Interestingly, it does work correctly if I directly type "ctrl-n" to do the completion.

Also, if I first use vim-latex to do tab completion of a \cite or a \ref -- which works correctly and is faster than Latex-Box -- then the tab completion of words will work correctly from that point forward.  This means that "exam" would now display a box for words like "example" and "examination".

Did I configure something incorrectly? Any advice that you can provide is appreciated!

**Comment by gkapfham** [comment]
As a follow on note, I wanted to let everyone know that I am no longer using SuperTab and instead I have migrated to exclusively using YouCompleteMe. Following the instructions in the vimtex user manual (that has been substantially updated since the time that I submitted this issue), I have been able to get it to work correctly for all of the important situations in which I need to complete reference names or labels. Thanks again for all of your help!
